docs/src/modules/components/AppLayoutDocs.js

```diff
@@ -156,7 +156,11 @@ export default function AppLayoutDocs(props) {
         }}
       />
       <AdManager {...(hasTabs && { classSelector: '.component-tabs' })}>
-        <Head title={`${title} - ${productName}`} description={description} card={card} />
+        <Head
+          title={`${title}${productName ? ` - ${productName}` : ''}`}
```

Comment:
This doesn't prevent the `console.error` in line 142 since the versions page doesn't have a `productName`. I don't want to change this logic much because the versions page is subject to change.